### PR TITLE
add catalog and managed acbs

### PR DESCRIPTION
**Pull request by steph-lieu**
add enablement of catalog and managed acbs playbook

### CODE REVIEW

**Comment by itsBryantP** [comment]
what are the minimum versions for dependencies required for this sample? Those should be documented 
- ansible-core
- IMS collection 
- z/OS Core collection
- ZOAU 
- PYZ

**Comment by steph-lieu** [comment]
> 

They are added to the roles readme, it needs to be added to the main page? These are what I used when I was working with core 1.4.0 and IMS collection 1.2.0

    ansible-core: 2.11.1
    IMS collection: 1.2.0
    z/OS Core collection: 1.4.0
    ZOAU: 1.1.1
    PYZ: 3.8

**Comment by itsBryantP** [comment]
> > 
> 
> They are added to the roles readme, it needs to be added to the main page? These are what I used when I was working with core 1.4.0 and IMS collection 1.2.0
> 
> ```
> ansible-core: 2.11.1
> IMS collection: 1.2.0
> z/OS Core collection: 1.4.0
> ZOAU: 1.1.1
> PYZ: 3.8
> ```
Yes can we document this on the top level readme for this sample.
IMS 1.2.0 states support for Python 3.9+ only. Can the required Python version be updated also?

File: zos_subsystems/ims/enable_catalog_macb/successful_macb_example.txt
Comment (itsBryantP) [inline zos_subsystems/ims/enable_catalog_macb/successful_macb_example.txt:1]: remove ec hostname in sample output and replace with generic hostname.

**Review by itsBryantP — CHANGES_REQUESTED** [review]

**Comment by itsBryantP** [comment]
Appears that https://github.com/stephanie-lieu is not active. Can we remove that user id from the support sections and only user @steph-lieu

**Comment by itsBryantP** [comment]
additional updates for review here https://github.com/steph-lieu/z_ansible_collections_samples/pull/1

File: zos_subsystems/ims/enable_catalog_macb/host_vars/zos_host.yml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+# The fully qualified dataset name for the IMS PROCLIB
+DFS_IMS_PROCLIB: '{{ DFS_AUTH_LIB_HLQ1 }}.{{ DFS_AUTH_LIB_HLQ2 }}.PROCLIB'
+
+# -------------------------- volume related settings ------------------------- #
+
+# first volume to use to store IMS datasets
+DFS_DS_VOLUME1: USER01
+# second volume to use to store IMS datasets
+DFS_DS_VOLUME2: USER01
```
[the remark this answers is not in the document]
Comment (steph-lieu) [reply zos_subsystems/ims/enable_catalog_macb/host_vars/zos_host.yml:88]: removed the `DFS_DS_VOLUME2` from the file, `DFS_DS_VOLUME1` is used

File: zos_subsystems/ims/enable_catalog_macb/successful_macb_example.txt
Comment (steph-lieu) [reply zos_subsystems/ims/enable_catalog_macb/successful_macb_example.txt:1]: changed host name & target user & volume

**Review by itsBryantP — APPROVED** [review]

**Review by itsBryantP — APPROVED** [review]

**Pull request merged**